src/app/features/user/active-user/active-user.component.html

```diff
@@ -0,0 +1,15 @@
+<mat-card>
+    <mat-card-content>
+        <div *ngIf="loading" style="text-align: center;">
+            <h3>Please waiting for activating your account...</h3>
```

Comment:
Please wait while we activate your account...

Reply:
thanks @1ceb3rg